examples/nfts.ts

```diff
@@ -0,0 +1,171 @@
+import { $, alice, bob, Event as CapiEvent, Rune, ValueRune } from "capi"
+import { signature } from "capi/patterns/signature/nfts.ts"
+import { Nfts, Utility, WestmintLocal } from "zombienet/nfts.toml/collator/@latest/mod.js"
+import { Event } from "zombienet/nfts.toml/collator/@latest/types/pallet_nfts/pallet.js"
+import { MintType } from "zombienet/nfts.toml/collator/@latest/types/pallet_nfts/types.js"
+import { RuntimeEvent } from "zombienet/nfts.toml/collator/@latest/types/westmint_runtime.js"
+
+// The nfts pallet uses inverted bitflags; on means exclude and off means include.
+const DefaultCollectionSetting = {
+  TransferableItems: 1n << 0n,
+  UnlockedMetadata: 1n << 1n,
+  UnlockedAttributes: 1n << 2n,
+  UnlockedMaxSupply: 1n << 3n,
+  DepositRequired: 1n << 4n,
+  allOff: 0n,
+  allOn: 0b11111n,
+}
+const DefaultItemSetting = {
+  Transferable: 1n << 0n,
+  UnlockedMetadata: 1n << 1n,
+  UnlockedAttributes: 1n << 2n,
+  allOff: 0n,
+  allOn: 0b111n,
+}
+
+// Create a collection
+const createCollection = await Nfts
+  .create({
+    config: Rune.rec({
+      settings: DefaultCollectionSetting.allOn,
+      mintSettings: Rune.rec({
+        mintType: MintType.Issuer(),
+        defaultItemSettings: DefaultItemSetting.allOn,
+      }),
+    }),
+    admin: alice.address,
+  })
+  .signed(signature({ sender: alice }))
+  .sent()
+  .dbgStatus("Create Collection:")
+  .finalizedEvents()
+
+// Extract the collection's id from emitted events
+const collection = await createCollection
+  .into(ValueRune)
+  .map(getCollectionIdFromEvents)
+  .unhandle(undefined)
+  .dbg("Collection Id:")
+  .run()
+
+const item = 0
+
+// Mint an item to the collection
+await Nfts
```

Comment:
Do you mean to run this? Is this why the example fails with an unhandled error?

Reply:
Yep!

Reply:
Adding the run to both fixes the error